src/rdkafka.h

```diff
@@ -7337,8 +7629,209 @@ RD_EXPORT const rd_kafka_topic_partition_list_t *
+ *
+ * @remark The result event type emitted on the supplied queue is of type
+ *         \c RD_KAFKA_EVENT_DESCRIBEGROUPS_RESULT
+ */
+RD_EXPORT
+void rd_kafka_DescribeGroups(rd_kafka_t *rk,
+                             const char **groups,
+                             size_t groups_cnt,
+                             const rd_kafka_AdminOptions_t *options,
+                             rd_kafka_queue_t *rkqu);
+
+/**
+ * @brief Get an array of group results from a DescribeGroups result.
+ *
+ * The returned groups life-time is the same as the \p result object.
+ *
+ * @param result Result to get group results from.
+ * @param cntp is updated to the number of elements in the array.
+ */
+RD_EXPORT
+const rd_kafka_ConsumerGroupDescription_t **
+rd_kafka_DescribeGroups_result_groups(
+    const rd_kafka_DescribeGroups_result_t *result,
+    size_t *cntp);
+
+
+/**
+ * @brief Gets the group id for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The group id, or NULL if \p desc is NULL.
+ */
+RD_EXPORT
+const char *rd_kafka_ConsumerGroupDescription_group_id(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Gets the error for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The group description error, or NULL if no error or \p desc is NULL.
+ */
+RD_EXPORT
+const rd_kafka_error_t *rd_kafka_ConsumerGroupDescription_error(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Is the \p desc group a simple consumer group.
+ *
+ * @param desc The group description.
+ * @return 1 if the group is a simple consumer group,
+ *         else 0 (also if \p desc is NULL).
+ */
+RD_EXPORT
+int rd_kafka_ConsumerGroupDescription_is_simple_consumer_group(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+
+/**
+ * @brief Gets the partition assignor for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The partition assignor, or NULL if \p desc is NULL.
+ */
+RD_EXPORT
+char *rd_kafka_ConsumerGroupDescription_partition_assignor(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+
+/**
+ * @brief Gets state for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return A group state, or RD_KAFKA_CGRP_STATE_UNKNOWN if \p desc is NULL.
+ */
+RD_EXPORT
+rd_kafka_consumer_group_state_t rd_kafka_ConsumerGroupDescription_state(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Gets the coordinator for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The group coordinator, or NULL if \p desc is NULL.
+ */
+RD_EXPORT
+rd_kafka_Node_t *rd_kafka_ConsumerGroupDescription_coordinator(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Gets the members count of \p desc group.
+ *
+ * @param desc The group description.
+ * @return The member count, or 0 if \p desc is NULL.
+ */
+RD_EXPORT
+int rd_kafka_ConsumerGroupDescription_member_cnt(
```

Comment:
you can spell this out: _count(), since these APIs are generally quite expressive.

Reply:
done